### ISSUE TITLE
Error 405, bot does not work.

### ISSUE THREAD

**Issue by mbledkowski** [Wildric-Auric/MidJourney-Wrapper#9]
When I run the bot, I get the following error
```
Ignoring exception in on_connect
Traceback (most recent call last):
  File "/home/mble/.local/lib/python3.10/site-packages/discord/client.py", line 352, in _run_event
    await coro(*args, **kwargs)
  File "/home/mble/.local/lib/python3.10/site-packages/discord/bot.py", line 793, in on_connect
    await self.register_commands()
  File "/home/mble/.local/lib/python3.10/site-packages/discord/bot.py", line 460, in register_commands
    await self.http.bulk_upsert_command_permissions(
  File "/home/mble/.local/lib/python3.10/site-packages/discord/http.py", line 338, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 405 Method Not Allowed (error code: 0): 405: Method Not Allowed
Logged in as h4xx3d midjourney#1614
```
Bot is online, but is unable to generate anything:
![image](https://user-images.githubusercontent.com/23295125/203211280-61678e24-366b-4fa2-b87c-68a46e9414e1.png)

**Comment by Wildric-Auric** [comment]
Hi, like described in [issue#7](https://github.com/Wildric-Auric/MidJourney-Wrapper/issues/7), try printing response.text to get more info.

**Comment by mbledkowski** [comment]
@Wildric-Auric Where exactly should i put this print statement?

**Comment by Wildric-Auric** [comment]
async def mj_imagine(ctx, prompt: discord.Option(str)):
    response = PassPromptToSelfBot(prompt)

**Comment by Wildric-Auric** [comment]
It's fixed now, look at #11.
